doc/yjit/yjit.md

```diff
@@ -166,7 +166,8 @@ The machine code generated for a given method can be printed by adding `puts Rub
 YJIT supports all command-line options supported by upstream CRuby, but also adds a few YJIT-specific options:
 
 - `--yjit`: enable YJIT (disabled by default)
-- `--yjit-exec-mem-size=N`: size of the executable memory block to allocate, in MiB (default 48 MiB)
+- `--yjit-mem-size=N`: soft limit of all memory bytes for YJIT in MiB (default: 128). Tries to limit `code_region_size + yjit_alloc_size`
```

Comment:
```suggestion
- `--yjit-mem-size=N`: soft limit on total YJIT memory consumption in MiB (default: 128). Tries to limit `code_region_size + yjit_alloc_size`
```

In the original, the "bytes" part read as a bit redundant with MiB to me.

Reply:
Maybe something short like "soft limit on YJIT memory usage in MiB"